storage/sql/crud.go

```diff
@@ -152,14 +152,15 @@ func (c *conn) UpdateAuthRequest(id string, updater func(a storage.AuthRequest)
 				claims_user_id = $9, claims_username = $10, claims_email = $11,
 				claims_email_verified = $12,
 				claims_groups = $13,
-				connector_id = $14, connector_data = $15,
-				expiry = $16
-			where id = $17;
+			    claims_locale = $14,
```

Comment:
indentation looks a bit off here

Reply:
weird, it looks normal on my IDE and gofmt ran clean.  Ill move it up a line and see if it aligns properly...